java/compaction/compaction-core/src/main/java/sleeper/compaction/job/commit/CompactionJobIdAssignmentCommitRequestSerDe.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2022-2024 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.compaction.job.commit;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import sleeper.compaction.job.CompactionJob;
+import sleeper.compaction.job.CompactionJobJsonSerDe;
+import sleeper.core.statestore.CommitRequestType;
+import sleeper.core.util.GsonConfig;
+
+public class CompactionJobIdAssignmentCommitRequestSerDe {
```

Comment:
For the other request types we have approval tests for the JSON generated by the serialiser. Please can we add something similar for this? Please see `CompactionJobCommitRequestSerDeTest`.

Reply:
Done, thanks